content/3.pecan/1.overview/2.methods-and-data.md

```diff
@@ -153,20 +153,20 @@ Example data from 790 tumor samples can be found in [Figure 5 of McLeod et al](h
 
 ### t-SNE Methods
 
-[RAPID RNA-Seq workflow](https://university.stjude.cloud/docs/genomics-platform/workflow-guides/rapid-rnaseq/) was used to generate RNA-seq expression count data and visualized via ProteinPaint's scatter plot.
+St. Jude Cloud's [Rapid RNA-Seq workflow](https://university.stjude.cloud/docs/genomics-platform/workflow-guides/rapid-rnaseq/) was used to generate RNA-seq expression counts data and visualized on PeCan's expression t-SNE plot.
```

Comment:
This is better, but needs updating as in #69. The Rapid RNA-Seq workflow is a different workflow and it doesn't produce count data.

Reply:
Let's just update this content here.